articles/welcome.md

```diff
@@ -1,62 +1,56 @@
-|  IQ# | IQ# (pronounced i-q-sharp) is an extension primarily used by Jupyter and Python to the .NET Core SDK that provides the core functionality for compiling and simulating Q# operations. |
+## Code first
 
-## Quantum Development Kit Documentation
+Dive into a 5-minute [quickstart](xref:microsoft.quantum.write-program) to code a Bell state.
 
-The current documentation includes the following topics.
+Implement [Grover's search algorithm](xref:microsoft.quantum.quickstarts.search) in just a few lines of code.
 
-* [Microsoft Quantum Development kit release notes](xref:microsoft.quantum.relnotes).
-* [Quantum computing concepts](xref:microsoft.quantum.concepts.intro) includes topics such as the relevance of linear algebra to quantum computing, the nature and use of a qubit, how to read a quantum circuit, and more.
-* [Getting Started with the Quantum Development Kit](xref:microsoft.quantum.install) describes how to quickly set up your quantum development environment from any platform (Windows 10, Linux, and Mac) using Visual Studio, Visual Studio Code or the command line.  
-* [Quickstart - your first quantum program](xref:microsoft.quantum.write-program) walks you through how to write an application that creates a quantum entanglment state in the Visual Studio development environment. You'll learn how to define a Q# operation, call the Q# operation using C#, and how to execute your quantum algorithm.
-* [Managing quantum machines and drivers](xref:microsoft.quantum.machines) describes how quantum algorithms are executed, what quantum machines are available, and how to write a non-Q# driver for the quantum program.
-* [Quantum development techniques](xref:microsoft.quantum.techniques.intro) specifies the core concepts used to create quantum programs in Q#. Topics include file structures, operations and functions, working with qubits, and some advanced topics.
-* [Q# standard libraries](xref:microsoft.quantum.libraries.standard.intro) describes the operations and functions that support both the classical language control requirement and the Q# quantum algorithms. Topics include control flow, data structures, error correction, testing, and debugging. 
-* [Q# language reference](xref:microsoft.quantum.language.intro) details the Q# language including the type model, expressions, statements, and compiler use.
-* [For more information](xref:microsoft.quantum.more-information) contains specially selected references to deep coverage of quantum computing topics.
-* [Quantum trace simulator reference](xref:Microsoft.Quantum.Simulation.Simulators.QCTraceSimulators) contains reference material about trace simulator entities and exceptions.
-* [Q# library reference](xref:microsoft.quantum.standardlibsintro) contains reference information about library entities by namespace.
+Try one of the [Quantum Katas](xref:microsoft.quantum.overview.katas), short training exercises to teach you how to program in Q# and learn quantum computing.
```

Comment:
I wonder if we might want to include a note here about running the Katas online in Jupyter... Something at the end of the sentence like "You can try the Quantum Katas directly in your browser here [link to aka.ms/try-quantum-katas]". I know this link is present on the Katas page that is linked to here, but only at the end of the article. This might showcase it more?

Reply:
Added